doc/z-chap5.xml

```diff
@@ -33,7 +33,13 @@
     <#Include Label="IsDirectedTree">
     <#Include Label="IsUndirectedTree">
     <#Include Label="IsEulerianDigraph">
+    <#Include Label="IsHamiltonianDigraph">
+    <#Include Label="HamiltonianPath">
     <#Include Label="IsCycleDigraph">
   </Section>
-  
+
+  <Section><Heading>Partial Orders</Heading>
+    <#Include Label="IsPartialOrderDigraph">
+    <#Include Label="IsMeetSemilatticeDigraph">
+  </Section>
```

Comment:
This section was deleted, you need to rebase on the current HEAD of the master branch.